spirit/category/admin/templates/spirit/category/admin/index.html

```diff
@@ -15,8 +15,8 @@ <h1 class="headline">{% trans "Category list" %}</h1>
     <div class="rows">
 
         {% for c in categories %}
-            <div class="row">
-                <a href="{{ c.get_absolute_url }}">{{ c.title }}</a>
+            <div class="row" style="background-colour: {{ c.colour }}">
+                <a {% if c.colour != '' %}style="color: {{ c.colour }}"{% endif %} href="{{ c.get_absolute_url }}">{{ c.title }}</a>
```

Comment:
Yeah, empty string, None, zero, empty collection, False. But check the comment for the model field, I prefer nullable charfields.